### DISCUSSION TITLE
Adding include path

### DISCUSSION THREAD

**Discussion by faveoled** [indoorvivants/sn-bindgen#152] [Q&A] [answered]
I'm getting the following error:
```
[bindgen] info Using following clang flags
      -I/usr/lib/llvm-14/lib/clang/14.0.0/include
      -I/usr/include/x86_64-linux-gnu
      -I/usr/include
[bindgen] error clang: fatal error: 'glib.h' file not found
```
Is there a way to add addditional clang path when using sn-bindgen?
Currently I have the following:
```scala
nativeConfig ~= { c =>
  c.withCompileOptions(c.compileOptions :+ "-I/usr/include/glib-2.0")
}
```

**Comment by keynmol** [comment, marked as answer]
The flags that you add to `nativeConfig` are what Scala Native uses to compile _your_ code.

But when you use bindgen, is uses libclang - which is an interface to Clang, which needs to be configured separately. 
Basically bindgen is invoking mini-compiler, so it also needs all the paths that regular clang would need.

This is a long way of saying that `Binding` has a `clangFlags` parameter which is just a list of strings that will be passed to clang used by bindgen. 

See example here: https://github.com/indoorvivants/sn-bindgen-examples/blob/main/build.sbt#L225
